deployment/ansible/group_vars/all

```diff
@@ -1,3 +1,5 @@
 ---
+develop: "'development' in group_names"
```

Comment:
Currently in the process of replacing these with custom filters in NYC because I could not get them to evaluate deterministically:

- https://github.com/azavea/nyc-trees/commit/a720e1ef98731f2626b19bdfc8a88a905eb24abe
- https://github.com/azavea/nyc-trees/commit/8fbdcd3d6ef2c690854482167f4215841df040ea
- https://github.com/azavea/nyc-trees/commit/34fec803a1ca4c673341a466d9b34474a9192d27
- https://github.com/azavea/nyc-trees/commit/e52fd15d15c59f2c4b5d13f27d72c81c646d6d5d
- https://github.com/azavea/nyc-trees/commit/e5d3b3f02430d8d7253b143c116bbc3be259017d
- https://github.com/azavea/nyc-trees/commit/1060f4ac3f346d02d2b12807ca2b92dfbc78aee2
- https://github.com/azavea/nyc-trees/commit/6406e4c25cc0592bc9b34fc1fd36aa5e0c492159